engine/lite/components/kafka/src/main/scala/pl/touk/nussknacker/engine/lite/components/LiteKafkaComponentProvider.scala

```diff
@@ -34,10 +34,10 @@ class LiteKafkaComponentProvider extends ComponentProvider {
   override def create(config: Config, dependencies: ProcessObjectDependencies): List[ComponentDefinition] = List(
     ComponentDefinition("kafka-json", new KafkaSinkFactory(GenericJsonSerialization(_), dependencies, LiteKafkaSinkImplFactory)),
     ComponentDefinition("kafka-json", new KafkaSourceFactory[String, java.util.Map[_, _]](
-      ConsumerRecordDeserializationSchemaFactory.fixedValueDeserialization(deserializeToMap), jsonFormatterFactory, dependencies, new LiteKafkaSourceImplFactory)),
```

Comment:
What impact have this change? Why it is done other way than in `FlinkKafkaComponentProvider`?

Reply:
It turned out it's actually not needed, I reverted this change...